db/migrate/20160709102546_add_username_to_user.rb

```diff
@@ -0,0 +1,30 @@
+class AddUsernameToUser < ActiveRecord::Migration
+  def self.up
+    add_column :users, :username, :string
+    User.find_each do |user|
+      self.update_username(user)
+    end
+    change_column_null :users, :username, false, ''
+    add_index :users, :username, unique: true
+  end
+
+  def self.update_username(user)
+    username = user.email
+    username = username.match(/(.+?)@(.+?)/)
+    username = username[1]
+    puts username
+    while true do
```

Comment:
__[rubocop]__
* Style/InfiniteLoop: Use `Kernel#loop` for infinite loops. [:ref](https://github.com/bbatsov/ruby-style-guide#infinite-loop)
* Style/WhileUntilDo: Do not use `do` with multi-line `while`. [:ref](https://github.com/bbatsov/ruby-style-guide#no-multiline-while-do)